cmd/deployment.go

```diff
@@ -134,6 +135,34 @@ func newDeploymentUpdateCmd(client *houston.Client, out io.Writer) *cobra.Comman
 	return cmd
 }
 
+func newDeploymentUserRootCmd(client *houston.Client, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "user",
+		Short: "Manage deployment user resources",
+		Long:  "Users can be added or removed from deployment",
+	}
+	cmd.AddCommand(
+		newDeploymentUserAddCmd(client, out),
+	)
+	return cmd
+}
+
+func newDeploymentUserAddCmd(client *houston.Client, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add EMAIL",
+		Short: "Add a user to a deployment",
+		Long:  "Add a user to a deployment",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
could you please add example similar to https://github.com/astronomer/astro-cli/blob/8ee42debef6220dea7204bac9118d74eb094a30d/cmd/deployment.go#L24